apps.md

```diff
@@ -65,4 +68,16 @@ layout: default
     {% endfor %}
   </table>
 
+  <div style="width: 100%; padding: 20px">
+    <hr style="height:5px; background-color:#00A4BD; border-width:0;">
+  </div>
+
+  {% include app_table.html apps=site.apps ds_type="raw" datatype="anat" %}
+  {% include app_table.html apps=site.apps ds_type="raw" datatype="func" %}
+  {% include app_table.html apps=site.apps ds_type="raw" datatype="dwi" %}
+
+  {% include app_table.html apps=site.apps ds_type="derivative" datatype="anat" %}
+  {% include app_table.html apps=site.apps ds_type="derivative" datatype="func" %}
+  {% include app_table.html apps=site.apps ds_type="derivative" datatype="dwi" %}
```

Comment:
so when we get an app for e.g., the eeg datatype, we'll have to add a line here (for the first time only), right?

Reply:
yup

actually there is already an app that can ingest "derivatives eeg" BIDS data (whatever that means)...

so we could already have it